src/turplanlegger/sql/crud.py

```diff
@@ -56,3 +58,53 @@ def update_user(db: Session, db_user: User, user_updates: UserUpdate) -> User |
         db.refresh(db_user)
 
     return db_user
+
+
+# Notes
+def delete_all_notes() -> None:
+    with Session(engine) as session:
+        statement = delete(Note)
+        session.exec(statement)  # type: ignore [call-overload]
+        session.commit()
+
+
+def create_note(db: Session, note: NoteCreate) -> Note | None:
+    db_note = Note.model_validate(note)
+    db.add(db_note)
+    db.commit()
+    db.refresh(db_note)
+    return db_note
+
+
+def get_all_notes(db: Session) -> Sequence[Note]:
+    statement = select(Note)
+    return db.exec(statement).all()
+
+
+# Remeber to only deliver notes beloging to the requesting user
```

Comment:
Eierskap-sjekk gjelder kanskje både for henting og update/delete også?

Reply:
Det stemmer!